File: backend-dotnet/Application/Commands/Auth/RegisterUserCommand.cs
using Application.DTOs;
using Domain.Entities;

namespace Application.Commands.Auth
{
    public class RegisterUserCommand : ICommand<(string token, User user)>
    {
        public RegisterDto Dto { get; set; }
        
        public RegisterUserCommand(RegisterDto dto)
        {
            Dto = dto;
        }
    }
}

File: backend-java/src/main/java/com/jerrygram/application/commands/auth/RegisterUserCommand.java
package com.jerrygram.application.commands.auth;

import com.jerrygram.application.dtos.RegisterDto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class RegisterUserCommand {
    private RegisterDto registerDto;
}